Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml

```
@@ -836,6 +836,17 @@ script:
       description: User-provided notes related to the incident.
       type: string
     description: Returns additional details about a specific incident.
+  - name: asm-get-external-websites
+    arguments:
+    - name: filter
```

```suggestion
    - name: authentication
```
In code this is really searching authentication, so you should change the argument to that instead of filter (similar to other commands)